### DISCUSSION TITLE
NsPanel eu homeassistant error

### DISCUSSION THREAD

**Discussion by maxtrin87** [Blackymas/NSPanel_HA_Blueprint#358] [Q&A]
Hello i am new to this just got myself a nspanel eu and followd marks guide on yt but i get stuck on Update TFT display it tock 24 hours to do and now all it  says is system data error, i have read and read but i dont get what i am doing wrong?
Please can someone help me?

**Comment by Blackymas** [comment]
I have answered you here
https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/359
